public/sass/_variables.dark.scss

```diff
@@ -100,27 +99,26 @@ $external-link-color: $sapphire-light;
 // Typography
 // -------------------------
 $headings-color: darken($white, 11%);
-$abbr-border-color: $gray-3 !default;
+$abbr-border-color: $gray-2 !default;
 $text-muted: $text-color-weak;
 
 $hr-border-color: $dark-4;
 
 // Panel
 // -------------------------
-$panel-bg: #212124;
+$panel-bg: $dark-3;
```

Comment:
this change does not feel great, makes panels a bit brighter and grayer and dashboard settings aside is affected.

Reply:
yes, I tried to merge dark-3. panel-bg and blue-gray into one color. Might have become to much compromising.